src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoFinalizerCheck.java

```diff
@@ -45,6 +45,31 @@
  * <pre>
  * &lt;module name=&quot;NoFinalizer&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *  public class Test {
+ *
+ *      protected void finalize() throws Throwable { //Violation
```

Comment:
Please change `//Violation` to `// violation`

Reply:
done